core/src/services/gcs/backend.rs

```diff
@@ -382,6 +382,7 @@ impl Accessor for GcsBackend {
                 read_can_next: true,
 
                 write: true,
+                write_with_cache_control: true,
```

Comment:
gcs returns object metadata on json, we need to load it:

https://cloud.google.com/storage/docs/json_api/v1/objects#resource

Visit the following code for more details:

https://github.com/apache/incubator-opendal/blob/d3be656ac018358f0382876107ef5b0805d03e8b/core/src/services/gcs/backend.rs#L557-L581

Reply:
Yep. I'm digging into docs.